requirements.txt

```diff
@@ -1,7 +1,4 @@
-opencv-python>=3.4.5.20
-pandas>=0.25.2
 torch>=1.8.0
 torchvision>=0.9.0
-tqdm>=4.20.0
```

Comment:
we are still using it in datasets/utils I think

Reply:
I already move datasets/utils to pyro-dataset as it's dataset creation related

Reply:
Not sure we talk about the same thing, but as far as this branch is concerned, there is still an import of tqdm : https://github.com/MateoLostanlen/pyro-vision/blob/remove_dataset_modules/pyrovision/datasets/utils.py#L11

Reply:
Yes we are, but this module will be deleted with openfire, I can remove the dependency in the other PR